graylog2-web-interface/packages/eslint-config-graylog/index.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
       rules: {
         'jest/expect-expect': [
           'error',
-          { assertFunctionNames: ['expect*', '(screen.)?find(All)?By*', 'selectEvent.assertOptionExists(*'] },
+          { assertFunctionNames: ['expect*', '(screen.)?find(All)?By*', 'selectEvent.assertOptionExists*'] },
```

Comment:
Just a thought, is it possible to specify this as `RegExp` instance instead of string? This way we would get earlier feedback (during loading of the package) that the syntax is invalid.